### PR TITLE
Added function generate_random_response

### PR DESCRIPTION
**Pull request by Ervin66**
response now returns with function call for cleaner code, returns a formatted string generated in generate_random_response function

    def generate_random_response(self):
        name = self.extract_name()
        stat = self.extract_stat()
        if not name:
            return inc_name
        elif not stat:
            return inc_stat

        stat_val = self.get_player_stat(name, stat)
        resp_1 = "Seems {} has a {} of {}.".format(name, stat, stat_val)
        resp_2 = "After checking my little black book, I've found that {} has {} under his name for {}.".format(name, stat_val, stat)
        resp_3 = "{}...obviously".format(stat_val)
        resp_4 = "Last I checked... {} had a {} of {}.".format(name, stat, stat_val)
        resp_5 = "Do you not have every NBA stat since the beginning of time memorized? Well he has a {} of {}.".format(stat, stat_val)
        resp_6 = "{} has {} of {}, impressive isn't it?".format(name, stat, stat_val)
        resp_7 = "As we are speaking, {} has {} of {}".format(name, stat, stat_val)
        resp_8 = "After watching hours of tapes, I came with {} of {} for {}".format(stat, stat_val, name)
        resp_9 = "If my calculations are correct {} has {} of {} during his career".format(name, stat, stat_val)
        resp_10 = "Couldn't you Google that? Well since we're already here: {} has {} of {}".format(name, stat, stat_val)
        resp_list = [resp_1,
                     resp_2,
                     resp_3,
                     resp_4,
                     resp_5,
                     resp_6,
                     resp_7,
                     resp_8,
                     resp_9,
                     resp_10]
        return random.choice(resp_list)

    def response(self):
        return self.generate_random_response(name, stat, stat_val)

### CODE REVIEW

**Comment by Ervin66** [comment]
I don't understand the errors

File: inference/statnode.py [hunk 1]
```diff
@@ -1,96 +1,113 @@
...
+class StatNode(object):
+
+    def __init__(self):
+        self.query = ""
+        self.nlp = spacy.load("en_core_web_md")
+
+    def load_query(self, query):
+        self.query = query
+
+    def generate_random_response(self, name, stat_val):
```
Comment (skekre98) [inline inference/statnode.py:20]: You are missing `stat` as an input parameter.

File: inference/statnode.py [hunk 2]
```diff
@@ -1,96 +1,113 @@
...
+                     resp_8,
+                     resp_9,
+                     resp_10]
+        return random.choice(resp_list)
+
+    def response(self):
+        name = self.extract_name()
+        stat = self.extract_stat()
+        stat_val = self.get_player_stat(name, stat)
+        if not name:
```
Comment (skekre98) [inline inference/statnode.py:47]: You need to move this before `get_player_stat` is called

File: inference/statnode.py [hunk 3]
```diff
@@ -1,96 +1,113 @@
...
+            return inc_name
+        elif not stat:
```
Comment (skekre98) [inline inference/statnode.py:49]: You need to move this before `get_player_stat` is called

**Comment by skekre98** [comment]
@Ervin66 see comments. There are a few issues about the way you have implemented `response`

**Review by skekre98 — APPROVED** [review]
LGTM!

**Pull request merged**